game-core/src/main/java/games/strategy/engine/lobby/client/ui/LobbyGamePanel.java

```diff
@@ -155,7 +161,10 @@ private void mouseOnGamesList(final MouseEvent e) {
 
     final JPopupMenu menu = new JPopupMenu();
 
-    getUserGamesListContextActions().forEach(menu::add);
+    Arrays.asList(
+            SwingAction.of("Join Game", this::joinGame),
+            SwingAction.of("Host Game", () -> hostGame(lobbyServerProperties)))
+        .forEach(menu::add);
```

Comment:
I don't think having a list makes this more readable at this point.

Reply:
Personally I think that says something about the helper method: ` getUserGamesListContextActions()`.
Having the values be one per line IMO is a readability win.

To get parity with before, we could extract the list to an intermediate variable, but I think that would not be much cleaner in the end and highlights the suspect extracted method (which was excessively indirect)